### ISSUE TITLE
Selecting multiple rows breaks when using templateCache

### ISSUE THREAD

**Issue by eh-am** [daniel-nagy/md-data-table#412]
I've noticed that when I use gulp-angular-templatecache, it transforms empty attributes such as **multiple** to **multiple="multiple"**, breaking multiple row selection.

The issue is in the function enableMultiSelect in mdTable.js:

```
    self.enableMultiSelect = function () {
      return $attrs.multiple === '' || $scope.$eval($attrs.multiple);
    };
```

The solution should be pretty simple, just checking if the attribute name is the same as the value, or even just checking the presence of the attribute.

**Comment by daniel-nagy** [comment]
Yeah, `multiple` is a boolean attribute and is part of the HTML spec. I am abusing it slightly. I'm working on a new version that should fix this.
